harness/determined/profiler.py

```diff
@@ -45,6 +45,15 @@ class ShutdownMessage:
     pass
 
 
+DEBUG = True
+
+
+def debug_log(*args: Any) -> None:
+    args_as_str = " ".join([str(arg) for arg in args])
+    if DEBUG:
+        logging.info(f"{LOG_NAMESPACE} (DEBUG) {args_as_str}")
```

Comment:
blocking: This is a bit weird. Don't harness set the logger level to debug if it's in debug mode? why do you use `logging.info` here?